docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+services:
+  web:
+    image: dockerimage:latest
+    ports:
+      - 5000:5000
+    networks:
+      - pacman-network
+
+  db:
+    image: mongo
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: root
+      MONGO_INITDB_ROOT_PASSWORD: rootpassword
```

Comment:
Not the most secure but not important for this case :D

Reply:
Agreed - I'll try and get this put into an env file, if possible :smile: